Tensile/Source/client/include/DataInitialization.hpp

```diff
@@ -57,6 +57,16 @@ namespace Tensile
             Count
         };
 
+        enum class BoundCheckMode
+        {
+            Disable = 0,
+            NaN,
+            GuardPageFront,
+            GuardPageEnd
```

Comment:
Should be either `Front`/`Back` or `Begin`/`End`.

Reply:
@sdquiring done